README.md

```diff
@@ -110,14 +111,25 @@ Technical writing is writing or drafting technical communication used in technic
 * [The Handbook of Technical Writing](https://www.amazon.com/gp/aw/d/1457675528/ref=cm_cr_arp_mb_bdcrb_top?ie=UTF8) - by Gerald J. Alre
 * [How To Write Usable User Documentation, 2nd Edition](https://www.amazon.com/How-Write-Usable-User-Documentation/dp/0897746392/ref=sr_1_1) - by Edmond H. Weiss
 * [Letting Go of the Words](https://www.goodreads.com/book/show/1135441.Letting_Go_of_the_Words) by Janice Redish
+* [Docs Like Code](https://www.amazon.com/Docs-Like-Code-Anne-Gentle/dp/1365816079) by Anne Gentle
+* [The Product is Docs](https://www.goodreads.com/book/show/37563319-the-product-is-docs) by Christopher Gales
+* [Every Page is Page One](https://www.amazon.com/Every-Page-One-Topic-Based-Communication/dp/1937434281) by Mark Baker
+
+## Style Guides
```

Comment:
I already included some style guide in the resource section and I think it would be great to make this a new category as you have suggested. Thank you :)